LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
+Rights to distribute the Mod must be granted in a separate license which must be explicitly obtained from the Author.
+
+If a user wishes others to aquire the mod, the user may provide a link to the Author's website. This doesn't need to be granted as a right. We're just letting you know the acceptable way your friends may download the Mod.
 
 6. Right to contribute
 --------
 
-The User may "fork" the Codebase.  
+The User may copy ("fork") the Codebase.  
 The User may compile the Codebase.  
-The User may submit contributions to the Author for inclusion in the Mod.  
-The User grants the Author all rights to any contribution.  
-The User renounces all rights to their contribution except as specified under this license.  
-The User retains the right to re-use code they've written.  
-The Author may specify a Contributor License Agreement allowing a Contributor to retain some or all rights.  
+The User may submit source code or other materials (a Contribution) to the Author to be considered for inclusion in the Mod.  
```

Comment:
I assumed the "Contribution" encompassed only the accepted portions of the submission. We should probably be more explicit about this. All the more reason to define "Contribution."

For reference, the publishing process for academic papers works this way. You only sign the copyright transfer form after the paper has been accepted. There's no point in transfering copyright when the paper is submitted.